### PR TITLE
Layer 2 translation

### PR DESCRIPTION
**Pull request by corwintines**
## Description
- Set layer 2 page up for translation

### CODE REVIEW

File: src/pages/layer-2.js [hunk 1]
```diff
@@ -284,92 +283,113 @@ const Layer2Page = ({ data }) => {
...
     },
   ]
 
   const toolsData = {
     information: [
       {
-        title: "L2BEAT",
-        description:
-          "L2BEAT is a great resource for looking at technical risk assessments of layer 2 projects. We recommend checking out their resources when researching specific layer 2 projects.",
+        title: translateMessageId("layer-2-tools-l2beat-title", intl),
```
Comment (wackerow) [inline src/pages/layer-2.js:336]: Typically we do not set product names up for translation. Some of these names, like "L2 Fees" I suppose are debatable, but many of these I would assume would remain untranslated, such as L2BEAT, Zepper, Zerion, etc.
image
Would suggest we revert these six strings.

File: src/pages/layer-2.js [hunk 2]
```diff
@@ -854,105 +795,68 @@ const Layer2Page = ({ data }) => {
...
           </p>
         </ExpandableCard>
       </PaddedContent>
 
       <PaddedContent>
-        <h2>Further reading</h2>
+        <h2>
+          <Translation id="layer-2-further-reading-title" />
+        </h2>
         <ul>
```
Comment (minimalsm) [inline src/pages/layer-2.js:860]: We should translate the further reading section

Screenshot 2022-05-30 at 15 39 42

File: src/pages/layer-2.js [hunk 3]
```diff
@@ -836,7 +775,9 @@ const Layer2Page = ({ data }) => {
       </PaddedContent>
 
       <PaddedContent>
-        <h2>Tools to be effective on layer 2</h2>
+        <h2>
+          <Translation id="layer-2-tools-title" />
+        </h2>
         <TwoColumnContent>
           <Flex50>
             <ProductList
```
Comment (minimalsm) [inline src/pages/layer-2.js:783]: We should queue up this JSON file for translation too.

Screenshot 2022-05-30 at 15 39 35

File: src/pages/layer-2.js [hunk 4]
```diff
@@ -258,19 +258,18 @@ const Layer2Page = ({ data }) => {
...
-        content: "What is layer 2",
+        content: translateMessageId("layer-2-hero-button-1", intl),
         pathId: "what-is-layer-2",
       },
       {
-        content: "Use layer 2",
+        content: translateMessageId("layer-2-hero-button-2", intl),
         pathId: "use-layer-2",
         isSecondary: "isSecondary",
       },
```
Comment (minimalsm) [inline src/pages/layer-2.js:275]: Oops missed this one 

Screenshot 2022-05-30 at 15 39 54

File: src/intl/en/page-layer-2.json [hunk 5]
```diff
@@ -2,5 +2,109 @@
   "layer-2-arbitrum-note": "Fraud proofs only for whitelisted users, whitelist not open yet",
   "layer-2-boba-note": "State validation in development",
   "layer-2-metis-note": "Fraud proofs in development",
-  "layer-2-optimism-note": "Fault proofs in development"
+  "layer-2-optimism-note": "Fault proofs in development",
+  "layer-2-hero-title": "Layer 2",
+  "layer-2-hero-header": "Ethereum for everyone",
+  "layer-2-hero-subtitle": "Scaling Ethereum without compromising on security or decentralization.",
+  "layer-2-hero-alt-text": "Layer 2 diagram",
```
Comment (minimalsm) [inline src/intl/en/page-layer-2.json:9]:
```suggestion
  "layer-2-hero-alt-text": "Illustration of transactions being rolled up on Layer 2 and posted to Ethereum Mainnet",
```

**Review by minimalsm — CHANGES_REQUESTED** [review]
Nice 😎💪

Left some comments on some small things that we missed adding to translations.

**Review by minimalsm — APPROVED** [review]

**Pull request merged**